api/src/controllers/application.js

```diff
@@ -66,7 +67,7 @@ router.post("/", passport.authenticate(["young", "referent"], { session: false,
     const { value, error } = validateNewApplication(req.body, req.user);
     if (error) return res.status(400).send({ ok: false, code: ERRORS.INVALID_PARAMS });
 
-    if (!value.hasOwnProperty("priority")) {
+    if ("priority" in value) {
```

Comment:
ducoup pardon mais c'est 
```suggestion
    if (!("priority" in value)) {
```